{"pyct.error_utils_test": ["platform.test.main"], "pyct.error_utils_test.ErrorMetadataBaseTest.test_create_exception_default_constructor": ["pyct.error_utils.ErrorMetadataBase", "test.TestCase.assertIn", "<builtin>.str", "test.TestCase.assertIsInstance"], "pyct.error_utils.ErrorMetadataBase": [], "test.TestCase.assertIsInstance": [], "<builtin>.str": [], "test.TestCase.assertIn": [], "pyct.error_utils_test.ErrorMetadataBaseTest.test_create_exception_custom_constructor": ["test.TestCase.assertIsNone", "pyct.error_utils.ErrorMetadataBase", "pyct.error_utils.ErrorMetadataBase"], "pyct.error_utils_test.ErrorMetadataBaseTest.test_create_exception_custom_constructor.CustomError.__init__": ["<builtin>.super"], "<builtin>.super": [], "test.TestCase.assertIsNone": [], "pyct.error_utils_test.ErrorMetadataBaseTest.test_get_message_no_code": ["pyct.error_utils.ErrorMetadataBase", "re.compile", "test.TestCase.assertRegex"], "re.compile": [], "test.TestCase.assertRegex": [], "pyct.error_utils_test.ErrorMetadataBaseTest.test_get_message_converted_code": ["re.compile", "test.TestCase.assertNotRegex", "test.TestCase.assertRegex", "pyct.error_utils.ErrorMetadataBase", "pyct.origin_info.OriginInfo", "pyct.origin_info.LineLocation"], "pyct.origin_info.LineLocation": [], "pyct.origin_info.OriginInfo": [], "test.TestCase.assertNotRegex": [], "pyct.error_utils_test.ErrorMetadataBaseTest.test_get_message_call_overload": ["pyct.error_utils.ErrorMetadataBase", "re.compile", "test.TestCase.assertRegex"], "platform.test.main": []}